tests/support_server/setup.pm

```diff
@@ -253,11 +253,15 @@ sub setup_xvnc_server {
 sub setup_xdmcp_server {
     return if $xdmcp_server_set;
 
+    if (check_var('REMOTE_DESKTOP_TYPE', 'xdmcp_xdm')) {
+        script_run "sed -i -e 's|^DISPLAYMANAGER=.*|DISPLAYMANAGER=\"xdm\"|' /etc/sysconfig/displaymanager";
+        script_run "sed -i -e 's|^DEFAULT_WM=.*|DEFAULT_WM=\"icewm\"|' /etc/sysconfig/windowmanager";
+    }
     script_run 'yast2 firewall services add zone=EXT service=service:xdmcp';
     script_run "sed -i -e 's|^DISPLAYMANAGER_REMOTE_ACCESS=.*|DISPLAYMANAGER_REMOTE_ACCESS=\"yes\"|' /etc/sysconfig/displaymanager";
     script_run "sed -i -e 's|^\\[xdmcp\\]|\\[xdmcp\\]\\nMaxSessions=2|' /etc/gdm/custom.conf";
     script_run 'systemctl restart display-manager.service';
-    assert_screen 'displaymanager';
+    assert_screen [qw(displaymanager displaymanager-xdm)];
```

Comment:
@okurz 
For xdmcp_gdm.pm the case will check displaymanager, for xdmcp_xdm.pm displaymanager-xdm will be checked